### PR TITLE
Theme and UI

### PR DESCRIPTION
**Pull request by psychoactiv**
- Created UI
- Added Routes
- Added Dark and Light Theme

https://user-images.githubusercontent.com/68146697/161124522-f2284d11-8f6a-45b3-87b4-df9291c8273d.mp4

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]
Overall the code looks really awesome.

File: src/components/Inputbox/input.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+.user-inp-btn-close {
+  background-color: var(--bg-color);
+  border: none;
+  font-size: 2rem;
+  cursor: pointer;
+  color: var(--grey);
+  padding: 5px;
+  border-radius: 999px;
```
Comment (dikshadhote) [inline src/components/Inputbox/input.css:48]: Use rem for values greater than 10px

File: src/components/sidebar/sidebar.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+.side-opt {
+  display: flex;
+  align-items: center;
+  height: 10rem;
+  padding: 2.5rem 1.8rem;
+  justify-content: flex-start;
+  column-gap: 2rem;
+  cursor: pointer;
+  border-radius: 0 20px 20px 0;
+  transition: 0.4s ease;
```
Comment (dikshadhote) [inline src/components/sidebar/sidebar.css:25]: use rem

File: src/components/sidebar/sidebar.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+.side-opt i {
+  font-size: 3rem;
+  padding: 1.5rem;
+  border-radius: 999px;
+  color: var(--icon-colors);
```
Comment (dikshadhote) [inline src/components/sidebar/sidebar.css:32]: use rem

**Review by dikshadhote — APPROVED** [review]
Overall code looks good , good to merge!

**Pull request merged**